backend/database/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
Add a link to Alembic. Also why is it generic?

Reply:
This is actually auto-generated by Alembic, so wanted to just leave it as is. Are you OK with that?